spec/formatting.md

```diff
@@ -963,4 +964,30 @@ The _Default Bidi Strategy_ is defined as follows:
          1. Append U+2069 POP DIRECTIONAL ISOLATE to `out`.
 1. Emit `out` as the formatted output of the message.
 
+> [!NOTE]
+> A _resolved value_ is derived from an _expression_
+> together with a _formatting context_.)
+> An implementation can use
+> a representation of _resolved values_ that tracks
+> everything needed to determine the directionality
+> of the formatted string representation
+> of a _resolved value_.
+> This can be accomplished by incorporating
+> the `isolate` flag used in step 2(iii),
+> as well as the separate directionality annotation
+> (one of « `'LTR'`, `'RTL'`, `'unknown'` »),
+> into the representation of _resolved values_.
+> Each _function handler_ can have its own means
+> for determining the directionality annotation
+> on the _resolved value_ it returns.
+> Alternately, an implementation could simply
+> determine directionality
+> based on the locale.
+
+> [!NOTE]
+> Directionality SHOULD NOT be determined by introspecting
+> the character sequence in the formatted string representation
+> of `resval`.
+
+
```

Comment:
We should not say SHOULD NOT in a non-normative note. If this is intended to be normative, it should be marked as `[!IMPORTANT]`, or not be a note.

```suggestion
Directionality SHOULD NOT be determined by introspecting
the character sequence in the formatted string representation
of `resval`.
```

Reply:
Changed to `[!IMPORTANT]`.